quest/src/api/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+target_sources(QuEST
+  PRIVATE
+  calculations.cpp
+  channels.cpp
+  debug.cpp
+  decoherence.cpp
+  environment.cpp
```

Comment:
random observation - `environment` in `environment.cpp` is being highlighted by Github as if it were a CMake keyword. Is this the case?? Should the filename be avoided? 😭